packages/sx05re/emuelec/bin/emuelecRunEmu.sh

```diff
@@ -110,6 +110,11 @@ if [[ "${EMULATOR}" = "retrorun" ]]; then
     LIBRETRO=""
 fi
 
+MIDI_OUTPUT="$(get_ee_setting "ra_midi_output" "${PLATFORM}" "${ROMNAME}")"
```

Comment:
it must be MIDI_OUTPUT=$(get_ee_setting "ra_midi_output" "${PLATFORM}" "${ROMNAME##*/}")  removing the quotation marks and using that ROMNAME parameter @Langerz82